website/events/migrations/0001_initial.py

```diff
@@ -7,7 +7,7 @@
 import django.db.models.deletion
 
 
-class Migration(migrations.Migration):
+class Migration(migrations.Migration): 
```

Comment:
I'm not sure why you edited this but you should definitely not commit it

Reply:
I am not sure why that was changed tbh.